pom.xml

```diff
@@ -267,6 +267,8 @@
                         <configuration>
                             <environmentVariables>
                                 <JBOSS_HOME>${jboss.home}</JBOSS_HOME>
+                                <JAEGER_SAMPLER_PARAM>1.0</JAEGER_SAMPLER_PARAM>
+                                <JAEGER_SAMPLER_TYPE>const</JAEGER_SAMPLER_TYPE>
```

Comment:
Hi @zhantemirov - I've read the comments in https://issues.redhat.com/browse/WFLY-13774, so IIUC this will be needed since the subsystem will no longer set those attributes by default and env variables will be needed for this, right?

Reply:
Yes.

The `standalone-microprofile.xml` configuration file (used by default when test suite is running) doesn't have Jaeger properties set by default anymore. As it's written in the WildFly MicroProfile documentation it is better to set these environmental variables to the lower values. I've defined these variables in Maven standard/default profile that uses `standalone-microprofile.xml`.

Whereas, as you already know, with `-Dmp.specs.scope=observability` option a `standalone.xml` configuration file is used - there is no need to set those variables as the OT subsystem already has these parameters set.

Reply:
I'd also like to hear a opinion from @jbliznak, thanks

Reply:
This solution is ok for me too.